deepspeed/autotuning/autotuner.py

```diff
@@ -499,7 +498,7 @@ def tune_space(self,
                    prev_best_mbs=0,
                    prev_best_metric_val=0):
         config_zero = tuning_space.get(ZERO_OPTIMIZATION, {})
-        stage = config_zero.get(ZERO_OPTIMIZATION_STAGE, ZERO_OPTIMIZATION_STAGE_DEFAULT)
+        stage = config_zero.stage
```

Comment:
Isn't this a bug if `config_zero` equals `{}`, and leads to evaluation of `{}.stage`?